share/jupyterhub/templates/home.html

```diff
@@ -26,10 +26,12 @@ <h2>
 
   <p>
   In addition to your default server,
-  you may have additional servers with names.
+  you may have additional {% if named_server_limit_per_user > 0 %}{{ named_server_limit_per_user }} {% endif %}server(s) with names.
   This allows you to have more than one server running at the same time.
   </p>
 
+  {% set named_spawners = user.all_spawners(include_default=False)|list %}
```

Comment:
Is there a reason this was created as a separate variable when its use hasn't changed?

Reply:
I was going to disable buttons to add new server based on the length of this list, but I realized it needs many changes in the JS. I think we can improve it in another PR, then we will need to use this variable.